sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -66,11 +66,13 @@ def __init__(self, endpoint, credential, **kwargs):
         # type: (str, Union[AzureKeyCredential, TokenCredential], Any) -> None
 
         authentication_policy = get_authentication_policy(credential)
+        polling_interval = kwargs.pop("polling_interval", POLLING_INTERVAL)
```

Comment:
Are we documenting this `polling_interval` kwarg to the client for users? I think a good idea could be to call it `_polling_interval` if we're the only people expecting the use it, but I don't feel too strongly about this

Reply:
This is an azure-core setting, so hopefully once Xiang documents all the core keywords we'll be able to link to that :)